boudicca.base/publisher-event-html/src/main/resources/templates/events/events.hbs

```diff
@@ -110,7 +120,11 @@
 <div style="text-align: center">
     <button id="loadMoreButton">Mehr laden</button>
 </div>
+<div id="resultNotFound" style="display: none">
+    <p>Keine Ereignisse gefunden für <span id="searchText"></span></p>
+    <p><a href="/" style="display: block">zurück zur Liste aller Ereignisse</a></p>
+</div>
 <div id="endOfResults" style="display: none">
-    <p>Du hast das Ende der Suchergebnisse erreicht.</p>
+    <p>Du haben das Ende der Ergebnisliste erreicht.</p>
```

Comment:
please revert it back to the original
1) it is Du hast not Du haben
2) Ergebnisliste is a clunky word in comparison with Suchergebnisse

Reply:
sorry to.bother you again with this :D but other than this the change looks good :)

Reply:
done.